src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDaoImpl actorDao;
+
+    public ActorServiceImpl(ActorDaoImpl actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElseThrow(() ->
+                new RuntimeException("Can't get Actor with id: " + id));
```

Comment:
Exception that occured on dao layer and is covered with DataProcessingException means some problems in communication with DB and undetermined result for get operation.
If dao returned empty optional without exception then it means that operation is successful but record is not present in DB
```suggestion
        return actorDao.get(id).orElseThrow(() ->
                new EnityNotFoundException("Actor with id: " + id + " was not found));
```